cypress/support/page_objects/sliderMove.js

```diff
@@ -0,0 +1,36 @@
+export class SliderMove {
+
+    movingSlider() {
```

Comment:
When we think about naming, we should try to imagine someone else (even non-technical person) or ourselves after few months/years reading our code: will we/he/she understand what this piece of code does? For example here: this method tests slider, so its name should be "testSliderByMovingIt". But what kind of tests it does? Does it move it left or right, or both? This makes name too long so it suggests that this method could be splitted into more . Also should tests be done inside Page Object or rather page object should be used to write tests?

For example: if we would call the class Slider, export it as a "slider" and name our methods moveLeft() and moveRight() then we would be able to write pretty nice tests elsewhere:

slider.moveLeft()
//check what happened
slider.moveRight()
//check what happened.

Reply:
Maybe I  change this part of the test but we have slider with two knots (min and max) and we move these left  and right - both of them. 
I don't sure enough your proposal ;-)